### DISCUSSION TITLE
Spinesever and The Night Blade should be contained in Strong Junkbox like in TBC.

### DISCUSSION THREAD

**Discussion by Apathyxia** [chromiecraft/chromiecraft#4792] [Suggestions]
In TBC, items [Spinesever](https://wowgaming.altervista.org/aowow/?item=34622) and [The Night Blade](https://wowgaming.altervista.org/aowow/?item=31331) were contained in [Strong Junkbox](https://wowgaming.altervista.org/aowow/?item=29569#contains). According to given links, they are not contained in WotLK. 

According to this [comment](https://www.wowhead.com/tbc/item=34622/spinesever#comments:id=187298) on Wowhead, it has been possible to acquire Spinesever from these boxes. 

According to this Youtube [video](https://youtu.be/yN4iRdZpFSs?t=488), The Night Blade has been acquired from the said junkbox. 

[Here](https://db.rising-gods.de/?item=29569#contains) is an another database to hint towards chances of acquiring the items in question.

**Comment by Gultask** [comment]
Oooh nice catch!
Open an issue (bug report) so that you can receive Chromie Points for this :)

TBCMangos has these items so we can port it from them:
![image](https://user-images.githubusercontent.com/100873791/211534915-4d0481d7-de04-44d8-b756-bf1a5a1e23b1.png)

**Reply by Apathyxia** [reply]
Thank you for your response. I have now opened an issue.
